### DISCUSSION TITLE
How can I add zooming?

### DISCUSSION THREAD

**Discussion by nandorojo** [glideapps/glide-data-grid#765] [Q&A]
What's the recommended way to handle zooming in/out like Google sheets? Should I just use `transform: scale()` on the parent? Not sure how the resolution will scale with this. 

I know this might be hard problem...curious to hear if there is a recommendation or built-in solution like a `zoom` prop.

Thanks!

**Comment by jassmith** [comment]
set `scaleToRem` to true and change the rem size of the screen/frame the grid is in.

**Reply by nandorojo** [reply]
ohh sweet i’ll try this out, thanks

**Reply by nandorojo** [reply]
will this work for `em` sizing too, so that i can style the parent rather than the `html` element where `rem` is set?

**Reply by jassmith** [reply]
Not at the moment, all doing that does anyway is change the font sizes and cell sizes and such. I will see about making it so you can directly pass a scalar.

**Reply by nandorojo** [reply]
much appreciated, that would be awesome as a prop
